JPS_Ontology/KBTemplates/TBox/OntoSewage.csv

```diff
@@ -0,0 +1,169 @@
+SewerageRecords,Class,,,,,,"the records such as functional state, ID, and owner of a sewerage component",https://www.theworldavatar.com/kg/ontosewage/,Sewerage records
+OwnershipType,Class,,,,,,defines the type of ownership for sewerage component,https://www.theworldavatar.com/kg/ontosewage/,Ownership type
+Public,Class,OwnershipType,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Public
+Private,Class,OwnershipType,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Private
+City,Class,OwnershipType,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,City
+FunctionalState,Class,,,,,,"is the operational state for sewerage component, i.e., under construction",https://www.theworldavatar.com/kg/ontosewage/,Functional state
+Expanded,Class,FunctionalState,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Expanded
+Stock,Class,FunctionalState,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Stock
+OutOfService,Class,FunctionalState,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Out of service
+Planning,Class,FunctionalState,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Planning
+Insulated,Class,FunctionalState,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Insulated
+ConstructionProperties,Class,,,,,,are the construction properties of a sewerage component such as construction year and materials used.,https://www.theworldavatar.com/kg/ontosewage/,Construction properties
+https://w3id.org/digitalconstruction/0.3/BuildingMaterials#Material,Class,,,,,,,https://w3id.org/digitalconstruction/0.3/BuildingMaterials,
+Concrete,Class,https://w3id.org/digitalconstruction/0.3/BuildingMaterials#Material,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Concrete
+PolyvinylChlorideHard,Class,https://w3id.org/digitalconstruction/0.3/BuildingMaterials#Material,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Polyvinyl chloride hard
+ReinforcedConcrete,Class,https://w3id.org/digitalconstruction/0.3/BuildingMaterials#Material,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Reinforced concrete
+Stoneware,Class,https://w3id.org/digitalconstruction/0.3/BuildingMaterials#Material,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Stoneware
+GrayCastIron,Class,https://w3id.org/digitalconstruction/0.3/BuildingMaterials#Material,IS-A,,,,,https://www.theworldavatar.com/kg/ontosewage/,Gray cast iron
+ChannelType,Class,,,,,,,https://www.theworldavatar.com/kg/ontosewage/,Channel type
+https://saref.etsi.org/saref4watr/Pipe,Class,SewerageComponent,IS-A,,,,,https://saref.etsi.org/saref4watr/,
+CrossSection,Class,,,,,,"a surface or shape exposed by making a straight cut through pipe, especially at right angles to an axis",https://www.theworldavatar.com/kg/ontosewage/,Cross section
+Shape,Class,,,,,,"is the external form of a Manhole's cover, flume, shaft, or cross section.",https://www.theworldavatar.com/kg/ontosewage/,Shape

```

Comment:
Some of them seem redundant, e.g. `circular` vs. `circle` vs `circle profile`. What's the difference?

![image](https://user-images.githubusercontent.com/79832739/204417443-5d63cd7a-75d6-4669-b511-abb33832c95f.png)

Reply:
Redundant sub-classes are removed.

Reply:
What's the difference between `round` and `circle`?

Reply:
Based on Cambridge dictionary:
round is shaped like a ball or circle, or curved.
circle is a continuous curved line, the points of which are always the same distance away from a fixed central point, or the area inside such a line.
Circle is definitely a 2-D. While ‘round' we can refer to 3-D.
But, I am not sure if here round is different with circle or not. Because, it can also refer to a circle.

Reply:
I guess we are not concerned about spheres here, but if the data contains both `round` and `circle`, then please go ahead keeping both.